proxy/envoy/testdata/envoy.json.golden

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
`go test istio.io/manager/proxy/envoy/...`
`cp manager/proxy/envoy/testdata/envoy.json manager/proxy/envoy/testdata/envoy.json.golden`

I can also add a flag to the test to save it as golden instead. Would that be better?